### ISSUE TITLE
Galleries "Follow Button" Contrast

### ISSUE THREAD

**Issue by JamieB-gu** [guardian/dotcom-rendering#5955]
The contrast on the follow button doesn't meet accessibility guidelines. Perhaps we should use the same text colours as are used for dark mode for now, since the background this text sits on is very similar to the dark mode background?

| News | Culture | Lifestyle |
| - | - | - |
| ![follow-news] | ![follow-culture] | ![follow-lifestyle] |

[follow-culture]: https://user-images.githubusercontent.com/53781962/188696243-9af2089e-1eee-47c5-b191-f4c2ee5ebd71.jpg
[follow-news]: https://user-images.githubusercontent.com/53781962/188696245-a8a8750d-d64d-4f36-990b-e1e4c9f591bf.jpg
[follow-lifestyle]: https://user-images.githubusercontent.com/53781962/188696246-a56251bd-bf06-499e-a783-a47926a853cb.jpg

**Comment by JamieB-gu** [comment]
Fixed in #5956
